src/components/ActivitiesListItem/ActivitiesListItem.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
 import { EDIT_ACTIVITY_ROUTE } from '../../constants/routes';
 import activityPropTypeShape from '../../prop-types/activity';
+import { spacing } from '../../styles/base/base';
 
-const ActivitiesListItem = ({ activity: { name, day, classNo, teacher, id, room } }) => (
-  <div>
-    <Link to={EDIT_ACTIVITY_ROUTE(id)} data-test="react-link">
-      <h3 data-test="activity-name">{name}</h3>
-    </Link>
-    <p data-test="activity-details">{`On ${day} on ${classNo} class with ${teacher} in room: ${room}`}</p>
-  </div>
+const ClassNoParagraph = styled.p`
+  display: inline;
+  margin: ${spacing.sSize};
+`;
+
+const EditActivityLink = styled(Link)`
+  word-break: break-all;
+`;
+
+const ActivityDescParagraph = styled.p`
+  margin: ${spacing.sSize};
+  word-break: break-all;
+`;
+
+const ActivitiesListItem = ({ activity: { name, classNo, teacher, id, room } }) => (
+  <>
+    <ClassNoParagraph>{`${classNo}.`}</ClassNoParagraph>
```

Comment:
Why not `ClassNoParagraph` ? Uncle Bob says, that we should avoid adding some reserved words to variable names like `Paragraph`. If there will be a change, that it shouldn't be a `paragraph`, you will have to change multiple places to be consistent. 
Also, I think that `ClassNo...` is not super name. Someone who doesn't know the project could think that this is negation. 
`Uncle Bob` -> Robert C. Martin